### ISSUE TITLE
Add option to disable automatic room join

### ISSUE THREAD

**Issue by co-robo** [coala/corobo#254]
Opened by @yukiisbored at [gitter//yukiisbored](https://gitter.im//yukiisbored?at=598af6f976a757f8088ab1fb)

**Comment by jayvdb** [comment]
@yukiisbored , I dont understand this issue.  is this a gitter specific problem?  I've created a label for gitter.

**Comment by yukiisbored** [comment]
Not sure if the problem still exists, but on startup corobo tries to join everyone's PM

**Comment by jayvdb** [comment]
@meetmangukiya , can you clarify if this is still a problem?

**Comment by meetmangukiya** [comment]
Yes, it joins everyone's PMs, else it won't be able to reply to users commands

**Comment by jayvdb** [comment]
Perhaps related to https://github.com/coala/corobo/issues/119 ?

**Comment by Makman2** [comment]
Can't it join PM rooms only when it's necessary?

**Comment by meetmangukiya** [comment]
How'd it determine someone messaged the bot unless it is listening to the private rooms? :)

**Comment by Makman2** [comment]
Doesn't it get a join notification if the user wants to go into a PM session with the bot?
